build/Dockerfile

```diff
@@ -103,6 +103,7 @@ FROM cuda-devel as python-installations
 ARG WHEEL_VERSION
 ARG USER
 ARG USER_UID
+ARG ENABLE_FMS_ACCELERATION=false
```

Comment:
For now I am going to set this as default false due to the error in accelerate_launch.py when trying to [load the tuned model](https://github.com/foundation-model-stack/fms-hf-tuning/blob/main/build/accelerate_launch.py#L158-L168) using `AutoModelForCausalLM.from_pretrained()` or `PeftModel.from_pretrained()`. When accelerate does their next release, we will no longer need to load the tuned checkpoint and can enable fms-acceleration to true by default